grades/admin.py

```diff
@@ -125,10 +125,32 @@ def has_add_permission(self, *agrs, **kwargs):  # pylint: disable=unused-argumen
     user_username.short_description = 'User'
 
 
+class MicromastersProgramCommendationAdmin(admin.ModelAdmin):
+    """Admin for MicromastersProgramCommendation"""
+    model = models.MicromastersProgramCommendation
+    list_display = ('id', 'user_username', 'program', 'uuid', 'created_on', 'updated_on')
```

Comment:
I think we should add `is_active` to the list display too since that might be something admins might be interested to know about on the list.